### ISSUE TITLE
Build used specific commit but tag was added to latest commit

### ISSUE THREAD

**Issue by itsho** [ReneSorensen/VSTSTools#34]
Hello.

I'm using "Git Tag" version 5.1.3, and found a possible bug:
when queuing a new build from a specific commit
I was expecting to have the tag on the said commit.

instead, the tag was added to the latest commit to the master branch.

```
2020-07-02T07:32:59.9131917Z ##[section]Starting: Git Tag commit
2020-07-02T07:32:59.9172700Z ==============================================================================
2020-07-02T07:32:59.9173299Z Task         : Git Tag
2020-07-02T07:32:59.9173802Z Description  : A simple task that tags a branch
2020-07-02T07:32:59.9174281Z Version      : 5.1.3
2020-07-02T07:32:59.9174743Z Author       : ATP P&I IT
2020-07-02T07:32:59.9175680Z Help         : tags the current branch with a specified tag. ### Prerequisites * Repository must be VSTS Git. * Allow scripts to access Oauth must be **Enabled** * Project Collection Build Service must have **Contribute** & **Create Tag** set to **Allow** or **Inherit Allow** for that particular repository
2020-07-02T07:32:59.9176714Z ==============================================================================
2020-07-02T07:33:00.2199015Z Setting working directory to 'E:\[redacted]'.
2020-07-02T07:33:01.0125580Z ##[command]git config user.email "[redacted]"
2020-07-02T07:33:01.0586059Z ##[command]git config user.name "[redacted]"
2020-07-02T07:33:01.0988857Z ##[command]git tag  "[redacted]"
2020-07-02T07:33:01.1538288Z Push tag to origin
2020-07-02T07:33:01.1540683Z ##[command]git push origin [redacted]
2020-07-02T07:33:04.8108917Z Locking support detected on remote "origin". Consider enabling it with:
2020-07-02T07:33:04.8109629Z   $ git config lfs.https://[redacted]/lfs.locksverify true
2020-07-02T07:33:04.8109971Z 
2020-07-02T07:33:04.8191428Z ##[section]Finishing: Git Tag commit

```

Possibly, related to #4 

Thank you very much in advance!

**Comment by VisualBean** [comment]
Will look into it

**Comment by VisualBean** [comment]
Fixed #36
